ipatests/test_integration/test_caless.py

```diff
@@ -126,26 +128,25 @@ def install(cls, mh):
             client_hostname = cls.clients[0].hostname
         else:
             client_hostname = 'unused-client.test'
-        cls.env = {
-            'domain': cls.master.domain.name,
-            'realm': cls.master.domain.name.upper(),
-            'server1': cls.master.hostname,
-            'server2': replica_hostname,
-            'client': client_hostname,
-            'dbdir': 'nssdb',
-            'dbpassword': cls.cert_password,
-            'crl_path': cls.crl_path,
-            'dirman_password': cls.master.config.dirman_password,
-        }
-        ipautil.run(['bash', '-ex', scriptfile], cwd=cls.cert_dir, env=cls.env)
+
+        create_caless_pki.domain = unicode(cls.master.domain.name)
+        create_caless_pki.realm = unicode(cls.master.domain.name.upper())
+        create_caless_pki.server1 = unicode(cls.master.hostname)
+        create_caless_pki.server2 = unicode(replica_hostname)
+        create_caless_pki.client = unicode(client_hostname)
+        create_caless_pki.password = unicode(cls.master.config.dirman_password)
+        create_caless_pki.cert_dir = unicode(cls.cert_dir)
+
+        # here we generate our certificates (not yet converted to .p12)
+        cls.log.info('Generating certificates to %s', cls.cert_dir)
+        create_caless_pki.create_pki()
 
         for host in cls.get_all_hosts():
             tasks.apply_common_fixes(host)
 
             # Copy CRLs over
-            base = os.path.join(cls.cert_dir, 'nssdb')
             host.transport.mkdir_recursive(cls.crl_path)
-            for source in glob.glob(os.path.join(base, '*.crl')):
+            for source in glob.glob(os.path.join(cls.crl_path, '*.crl')):
```

Comment:
If the module works the same as in freeipa-tools, then I believe the CRL files are to be found in `cls.cert_dir`